pkg/tests/apis/helper.go

```diff
@@ -43,33 +42,25 @@ type K8sTestHelper struct {
 	env        server.TestEnv
 	namespacer request.NamespaceMapper
 
-	Org1 OrgUsers
-	Org2 OrgUsers
+	Org1 OrgUsers // default
+	OrgB OrgUsers // some other id
 
 	// // Registered groups
 	groups []metav1.APIGroup
 }
 
-func NewK8sTestHelper(t *testing.T) *K8sTestHelper {
+func NewK8sTestHelper(t *testing.T, opts testinfra.GrafanaOpts) *K8sTestHelper {
 	t.Helper()
-	dir, path := testinfra.CreateGrafDir(t, testinfra.GrafanaOpts{
-		AppModeProduction: true, // do not start extra port 6443
-		DisableAnonymous:  true,
-		EnableFeatureToggles: []string{
-			featuremgmt.FlagGrafanaAPIServer,
-			featuremgmt.FlagGrafanaAPIServerWithExperimentalAPIs,
-		},
-	})
-
+	dir, path := testinfra.CreateGrafDir(t, opts)
 	_, env := testinfra.StartGrafanaEnv(t, dir, path)
 	c := &K8sTestHelper{
 		env:        *env,
 		t:          t,
 		namespacer: request.GetNamespaceMapper(nil),
 	}
 
-	c.Org1 = c.createTestUsers(int64(1))
-	c.Org2 = c.createTestUsers(int64(2))
+	c.Org1 = c.createTestUsers("Org1")
+	c.OrgB = c.createTestUsers("OrgB")
```

Comment:
^^ unable to actually create an org with id 2 -- need to create it by name

Reply:
> unable to actually create an org with id 2

could a test helper be added to orgs to make things more explicit, or would this be a big lift (or generally bad idea) because of how things are structured?

Reply:
obviously it is possible -- my fear is how big of a change it is.  The org/user/team stuff all has special flags/state for making test users.  

For our needs we want to test users in org 1 (so that namespace becomes "default" or "stack-XYZ") and users is a *different* org, the explicit ID does not really matter.